src/wallet/rpcwallet.cpp

```diff
@@ -2465,6 +2466,7 @@ static RPCHelpMan getwalletinfo()
     int64_t kp_oldest = pwallet->GetOldestKeyPoolTime();
     obj.pushKV("walletname", pwallet->GetName());
     obj.pushKV("walletversion", pwallet->GetVersion());
+    obj.pushKV("format", pwallet->GetDatabase().Format());
```

Comment:
Since it is related to the "descriptors" bool field, maybe place "format" at the end right after "descriptors".

Reply:
For now it's related. But note that you can have descriptor wallets in bdb format.